modules/xfem/test/include/kernels/TestMatTimeDerivative.h

```diff
@@ -0,0 +1,33 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#ifndef TESTMATTIMEDERIVATIVE_H
+#define TESTMATTIMEDERIVATIVE_H
+
+#include "TimeDerivative.h"
+
+// Forward Declaration
+class TestMatTimeDerivative;
```

Comment:
Is this kernel necessary here? Can we use rhoCp = 1 for all verification tests?

Reply:
As we discussed, I could adjust the timestep to maintain the analytical solution, but this test kernel is lightweight enough and out of the way (in test objects) that I think it's fine. It's something @bwspenc and I have spoken about and it seems to be okay.